khal/khalendar/vdir.py

```diff
@@ -224,10 +228,9 @@ def upload(self, item):
                 fpath, etag = self._upload_impl(item, href)
             else:
                 raise
-
         return href, etag
 
-    def _upload_impl(self, item, href):
+    def _upload_impl(self, item, href: str):
```

Comment:
**nitpick**: return type? I believe it's `Tuple[str, str]`.

Reply:
fixed